doc/tutorials/faq.md

```diff
@@ -0,0 +1,332 @@
+corner there is a link to [GeoHack](http://tools.wmflabs.org/geohack/geohack.php?pagename=Schladming&params=47_23_39_N_13_41_21_E_type:city(4565)_region:AT-6),
+which effectively tells you the coordinates are:
+
+    WGS84:
+    47° 23′ 39″ N, 13° 41′ 21″ E
+    47.394167, 13.689167
+
+So the next step would be to put the decimal coordinates into an array and use
+it as center:
+
+```javascript
+var schladming = [47.394167, 13.689167]; // caution partner, read on...
+// since we are using OSM, we have to transform the coordinates...
+var schladminWebMercator = ol.proj.fromLonLat(schladming);
+
+var map = new ol.Map({
+  layers: [
+    new ol.layer.Tile({
+      source: new ol.source.OSM()
+    })
+  ],
+  target: 'map',
+  view: new ol.View({
+    center: schladminWebMercator,
+    zoom: 9
+  })
+});
+```
+
+Running the above example will possibly surprise you, since we are not centered
+on Schladming, Austria, but instead on Abyan, a region in Yemen (possibly also a
+nice place). So what happened?
+
+Many people mix up the order of longitude and latitude in a coordinate array.
+Don't worry if you get it wrong at first, many OpenLayers developers have to
+think twice about whether to put the longitude or the latitude first when they
+e.g. try to change the map center.
+
+Ok, then let's flip the coordinates:
+
+```javascript
+var schladming = [13.689167, 47.394167]; // longitude first, then latitude
+// since we are using OSM, we have to transform the coordinates...
+var schladminWebMercator = ol.proj.fromLonLat(schladming);
+
+var map = new ol.Map({
+  layers: [
+    new ol.layer.Tile({
+      source: new ol.source.OSM()
+    })
+  ],
+  target: 'map',
+  view: new ol.View({
+    center: schladminWebMercator,
+    zoom: 9
+  })
+});
+```
+
+Schladming is now correctly displayed in the center of the map.
+
+So when you deal with EPSG:4326 coordinates in OpenLayers, put the longitude
+first, and then the latitude. This behaviour is the same as we had in OpenLayers
+2, and it actually makes sense because of the natural axis order in WGS84.
+
+If you cannot remember the correct order, just have a look at the method name
+we used: `ol.proj.fromLonLat`; even there we hint that we expect longitude
+first, and then latitude.
+
+
+## Why aren't there any features in my source?
+
+Suppose you want to load a KML file and display the contained features on the
+map. Code like the following could be used:
+
+```javascript
+var vector = new ol.layer.Vector({
+  source: new ol.source.KML({
+    projection: ol.proj.get('EPSG:3857'),
+    url: 'data/kml/2012-02-10.kml'
+  })
+});
+```
+
+You may ask yourself how many features are in that KML, and try something like
+the following:
+
+```javascript
+var vector = new ol.layer.Vector({
+  source: new ol.source.KML({
+    projection: ol.proj.get('EPSG:3857'),
+    url: 'data/kml/2012-02-10.kml'
+  })
+});
+var numFeatures = vector.getSource().getFeatures().length;
```

Comment:
I think it is an array: http://openlayers.org/en/v3.4.0/apidoc/ol.source.KML.html?unstable=true#getFeatures